TicketSwap/solar
# Update storybook to 6.x

# Description

This PR will make sure we can update storybook to 6.x. 

https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#from-version-53x-to-60x

## How to test

* Run `yarn`
* Run `yarn dev`
* Checkout storybook, verify that everything is working as expected
